backend/accounts/models.py

```diff
@@ -0,0 +1 @@
+"""Models"""
```

Comment:
No need to keep empty files, so we could remove it or add meaningful description? E.g.
```suggestion
"""Models for the accounts application."""
```
The same for the rest.